build/import/Packages.targets

```diff
@@ -94,8 +94,8 @@
     <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Query"                             Version="16.10.337-pre" />
 
     <!-- Roslyn -->
-    <PackageReference Update="Microsoft.VisualStudio.LanguageServices"                                Version="3.10.0-2.21121.14" />
-    <PackageReference Update="Microsoft.CodeAnalysis"                                                 Version="3.10.0-2.21121.14" />
+    <PackageReference Update="Microsoft.VisualStudio.LanguageServices"                                Version="4.0.0-2.21302.13" />
+    <PackageReference Update="Microsoft.CodeAnalysis"                                                 Version="4.0.0-2.21302.13" />
```

Comment:
@jasonmalinowski are these package versions appropriate for 16.11?

Reply:
I suspect not, based on the build errors around package downgrades. For example:

> NU1605: Detected package downgrade: Microsoft.VisualStudio.Threading from 17.0.15-alpha to 16.10.53-alpha

Reply:
No, we'll need to get you a 3.11 version. PR now open to create it: https://github.com/dotnet/roslyn/pull/53855